### PR TITLE
Handle first_page and last_page cases when reaching there.

### PR DESCRIPTION
**Pull request by longthanhtran**
While try vuejs-paginator component with yii2 rest, I found there are 2 cases where _first_page_ and _last_page_ can not handle properly. Hence my try is to check those 2 cases to ensure the component can work properly. Hopefully if this fits for other rest api(s)

### CODE REVIEW

File: src/VPaginator.vue [hunk 1]
```diff
@@ -65,8 +65,8 @@ export default {
     makePagination (data) {
       this.current_page = utils.getNestedValue(data, this.config.remote_current_page)
       this.last_page = utils.getNestedValue(data, this.config.remote_last_page)
-      this.next_page_url = utils.getNestedValue(data, this.config.remote_next_page_url)
-      this.prev_page_url = utils.getNestedValue(data, this.config.remote_prev_page_url)
+      this.next_page_url = (this.current_page === this.last_page) ? "" : utils.getNestedValue(data, this.config.remote_next_page_url);
```
Comment (hootlex) [inline src/VPaginator.vue]: Instead of `""` the url should equal `null` when there is not next/previous page.

**Comment by hootlex** [comment]
Hey @longthanhtran,
thanks for your contribution.

Why your API returns next/previous page urls when they don't exist? 
Maybe you are doing something wrong?

**Comment by longthanhtran** [comment]
Hi, since I use yii2 and it always return "no prev" on _first page_ or "no next" on _last page_. I think this is also reasonable as these case there should be optional. Otherwise, the component fails to render first / last list

File: src/VPaginator.vue (hunk 1 above)
Comment (longthanhtran) [inline src/VPaginator.vue]: thank you, I align the parts as your advice. Cheers

**Comment by hootlex** [comment]
@longthanhtran you are doing this check
```
this.current_page === this.last_page
```
But this doesn't solve the issued with 'no prev', right?

**Comment by longthanhtran** [comment]
the  `this.current_page === this.last_page` to handle last page where I don't need to worry about 'no prev'. On the other side, I only take care first page without worry about 'no next'. Both of the cases are checked :)

**Comment by hootlex** [comment]
won't `this.current_page === this.last_page` return false when last_page is 'no next'?

**Comment by longthanhtran** [comment]
I am sorry, seems I make you confuse. Here are the cases I checked.
* next_page_url. When I see the last page `this.last_page` then I simply use null instead of no value from API. (in my case, there is no last_page from API). For example, this.current_page is 10 and this.last_page also is 10, then the returned value to next_page_url is null.
```javascript
this.next_page_url = (this.current_page === this.last_page) ? null : utils.getNestedValue(data, this.config.remote_next_page_url);
```

* prev_page_url. Where the `current_page` is 1 then this should be the first page, I don't expect to have prev_page and I use null for this case. (in my case, there is no prev_page from API)
```javascript
this.prev_page_url = (this.current_page === 1) ? null : utils.getNestedValue(data, this.config.remote_prev_page_url);
```

**Comment by hootlex** [comment]
@longthanhtran oh I see now.
Thanks!

**Pull request merged**